components/Headers/__tests__/__snapshots__/AppHeader.test.js.snap

```diff
@@ -1,36 +1,48 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Component: AppHeader should render the default image in the component if no icon 1`] = `
+exports[`Component: AppHeader should handle missing info and show unknown if no data 1`] = `
 <div
-  class="sc-bczRLJ dIvpJ"
+  class="chakra-stack css-184jys2"
 >
+  <h2
+    class="chakra-heading css-0"
+  >
+    Connect
+  </h2>
   <div
-    class="sc-dkzDqf hDaBCz"
+    class="chakra-stack css-84zodg"
   >
-    <img
-      class="sc-hKMtZM eFtwgT"
-      src="/images/default-img.svg"
-    />
+    <p
+      class="chakra-text css-mwent8"
+    >
+      Unknown
+    </p>
   </div>
-  <div
-    class="sc-eCYdqJ hBueAj"
-  />
 </div>
 `;
 
 exports[`Component: AppHeader should render the the component with icon 1`] = `
 <div
-  class="sc-bczRLJ dIvpJ"
+  class="chakra-stack css-184jys2"
```

Comment:
Are these CSS classes stable?

Reply:
@justinbarry good catch. I added something to make the classes generated by emotion for this deterministic.